src/run/formlyCustomTags.js

```diff
@@ -3,22 +3,14 @@ export default addCustomTags
 
 // @ngInject
 function addCustomTags($document) {
-  if ($document && $document.get) {
-    // IE8 check ->
-    // http://stackoverflow.com/questions/10964966/detect-ie-version-prior-to-v9-in-javascript/10965203#10965203
+  if ($document && $document.documentMode < 9) {
     const document = $document.get(0)
-    const div = document.createElement('div')
-    div.innerHTML = '<!--[if lt IE 9]><i></i><![endif]-->'
-    const isIeLessThan9 = (div.getElementsByTagName('i').length === 1)
-
-    if (isIeLessThan9) {
       // add the custom elements that we need for formly
-      const customElements = [
-        'formly-field', 'formly-form', 'formly-custom-validation', 'formly-focus', 'formly-transpose',
-      ]
-      angular.forEach(customElements, el => {
-        document.createElement(el)
-      })
-    }
+    const customElements = [
+      'formly-field', 'formly-form', 'formly-custom-validation', 'formly-focus', 'formly-transpose',
```

Comment:
While you're at it, could you remove `formly-transpose`. I think when this was added, someone mistook `formly-transclude` for `formly-transpose`. But `formly-transclude` never actually makes its way to the DOM so we don't need to worry about that...

Also `formly-focus` and `formly-custom-validation` are attribute directives, so they shouldn't be in this list either.

Thanks for cleaning this up!